.azuredevops/pipelines/build-and-release.yml

```diff
@@ -44,10 +44,10 @@ stages:
         variables:
           - group: Code Sign KV Auth
 
-          - name: Configuration
-            value: Release
+           - name: ProjectFolder
```

Comment:
This change in indentation has broke the azure pipeline.

Reply:
This is fixed in #5